draft-xu-risav.md

```diff
@@ -223,8 +230,12 @@ Conversely, if any AS no longer publishes a `RISAVAnnouncement`, other ASes MUST
 
 > TODO: Discuss changes to the contact IP, check if there are any race conditions between activation and deactivation, IKEv2 handshakes in progress, SA expiration, etc.
 
+SA has its own expiration time and IKE has its keepalive mechanism. In abnormal case, i.e. the connection is failed after the IKE handshake is established, SA will be always in effect during its lifetime until it expires or the IKE keepalive is failed. In normal case, i.e. the connection is actively down, SA will be expired and RISAV will be disabled immediately.
```

Comment:
The IKE keepalive mechanism seems to be only for NAT traversal mode with UDP encapsulation, so I'm not sure that it applies here.

My real concern is about more complicated edge cases, such as when a network disables RISAV in the middle of a handshake, or when one network activates RISAV for the first time at the same time that another network is deactivating it.  If the networks disagree about whether RISAV is supposed to work, this could result in an outage (especially if we are "strict" to prevent any attacker from disabling RISAV).

Rather than trying to solve this here, I think we should open an issue or wait for input from the working group.